app.json

```diff
@@ -4,7 +4,7 @@
     "description": "This project is really great.",
     "slug": "campo-libre",
     "privacy": "public",
-    "sdkVersion": "25.0.0",
+    "sdkVersion": "24.0.0",
     "platforms": ["ios", "android"],
     "version": "1.0.0",
     "orientation": "portrait",
```

Comment:
(Apparently I'm not intelligent enough to figure out how to comment on images... so this comment is for the next two images below...) 
1) Renamed `intro.jpg` to `hero.jpg` as it seems clearer as to what it actually is
2) Replaced the white splashscreen with a temporary placeholder I found and cropped online